src/components/academies/AcademyDetailPage.jsx

```diff
@@ -1,34 +1,18 @@
-import { useMemo } from 'react';
 import { Breadcrumb, Container } from '@openedx/paragon';
 import { Link, useParams } from 'react-router-dom';
 import { FormattedMessage, useIntl } from '@edx/frontend-platform/i18n';
-import algoliasearch from 'algoliasearch/lite';
-import { getConfig } from '@edx/frontend-platform/config';
 import NotFoundPage from '../NotFoundPage';
 import './styles/Academy.scss';
 import { useAcademyDetails, useEnterpriseCustomer } from '../app/data';
 import AcademyContentCard from './AcademyContentCard';
 
 const AcademyDetailPage = () => {
-  const config = getConfig();
   const { academyUUID } = useParams();
   const { data: enterpriseCustomer } = useEnterpriseCustomer();
   const { data: academy } = useAcademyDetails();
   const academyURL = `/${enterpriseCustomer.slug}/academy/${academyUUID}/?${enterpriseCustomer.uuid}`;
   const intl = useIntl();
 
-  // init algolia index
-  const courseIndex = useMemo(
```

Comment:
Great idea to avoid prop drilling of the index, relying on direct usages of `useAlgoliaSearch` instead!